modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/sections/actions/ActionsInfo.js

```diff
@@ -64,20 +64,12 @@ const ActionsInfo = ({
 	if (functionActionExecutors?.length) {
 		actionTypeOptions.push(
 			...functionActionExecutors.map((item) => {
-				const itemCopy = {...item};
-				itemCopy.type = 'functionActionExecutor';
-				itemCopy.label = item.description;
-				delete itemCopy.description;
-				itemCopy.value = item.key;
-				delete itemCopy.key;
-
-				return Object.keys(itemCopy)
-					.sort()
-					.reduce((accumulator, key) => {
-						accumulator[key] = itemCopy[key];
-
-						return accumulator;
-					}, {});
+				const newItem = {};
+				newItem.label = item;
```

Comment:
@cgmonte we should add a blank line between the creation and usage of `newItem`.

Reply:
Yeah, totally, I changed it so now the [object is returned directly](https://github.com/liferay-workflow/liferay-portal/pull/845/commits/1aa4d1ec6ba459878aa2b158ff3e0d6265e25eaf#diff-bfaa66466473b228180534f19d15c76bc9fd0b9b13865f774016dd240dde4b98R66), without even creating a variable. Thanks for your observation, @naraandrade :)